Plugins/Administration/NWScript/nwnx_admin_t.nss

```diff
@@ -15,19 +8,19 @@ void main()
     string sString = "FuzzyKittens";
 
     NWNX_Administration_SetPlayerPassword(sString);
-    report("{Set/Get}PlayerPassword", NWNX_Administration_GetPlayerPassword() == sString);
+    NWNX_Tests_Report("NWNX_Administration", "{Set/Get}PlayerPassword", NWNX_Administration_GetPlayerPassword() == sString);
```

Comment:
Minor thing: Instead of "NWNX_{PluginName}" you could use the NWNX_{PluginName} constant defined in the plugin's include file